mlos_core/mlos_core/optimizers/flaml_optimizer.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, *,   # pylint: disable=too-many-arguments
         self._suggested_config: Optional[dict]
 
     def _register(self, configurations: pd.DataFrame, scores: pd.DataFrame,
-                  context: Optional[pd.DataFrame] = None) -> None:
+                  metadata: Optional[pd.DataFrame] = None) -> None:
```

Comment:
Please restore `context`.
The idea was to *add* `metadata` in addition to it, not replace it.

Reply:
(requires associated changes throughout)